proposals/evaluation/uc-3-inheritance.md

```diff
@@ -75,7 +75,7 @@ The acl enabling read access to all resources contained by `</weekly-status/>` f
 # Resource: </weekly-status/.acl>
 []
   acl:agentGroup </groups/research#g1> ;
-  acl:default <.> ;
+  acl:default <./> ;
```

Comment:
That comes to the same. I am not sure it is that much clearer. The dot in unix means `this directory` and
this is what it means here too (with `directory` replaced with `container` of course)

I tested that with cwm and [Easy RDF](https://www.easyrdf.org/converter)
```
@base <http://localhost/acl> .
@prefix acl: <https://ex.com/> .
[]
  acl:agentGroup </groups/research#g1> ;
  acl:default <.> ;
  acl:mode acl:Read .
```

Reply:
Ah cool, I prefer the `<.>` syntax personally, so if it's the same, I'd say let's revert.